README.rst

```diff
@@ -165,6 +165,16 @@ When the switch ``enabled`` is set to ``true``, HTTP Basic Authentication is ena
 
 If HTTP Basic Authentication is enabled then the CLI will send HTTP requests using HTTPS instead of HTTP.
 
+bndl
+^^^^
+
+The ``bndl`` command is used to create or modify bundles. It can be used for the following:
+
+- Creating a bundle from Docker and OCI images
```

Comment:
In the `Releasing` section we are using `*`. But you are right, both is fine. So let's leave it as it is.